packages/api/src/index.ts

```diff
@@ -16,7 +16,16 @@ export type {
   TrustSetFlags
 } from '@gemwallet/constants';
 
-// API return types
+// API request types
+export type {
+  GetNetworkRequest,
```

Comment:
@florent-uzio do you think these types should be renamed like that:
```suggestion
  GetNetworkRequest as GetNetworkParameter,
```